plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/WebXmlValidatorPluginTest.java

```diff
@@ -51,30 +52,30 @@
 
   @Before
   public void setUp() throws CoreException {
-    // Project's default source folder is the main project folder.
     IProject project = projectCreator.getProject();
     javaProject = projectCreator.getJavaProject();
 
     resource = project.getFile("WebContent/WEB-INF/web.xml");
 
-    ValidationTestUtils.createFolders(project, new Path("src/main/java"));
-    IFile servletClass = project.getFile("src/main/java/ServletClass.java");
```

Comment:
Are these files no longer used? If so they should be deleted in this PR. 

That said, including the test data as separate files rather than inline strings is much easier to edit and extend going forward. Is there anyway to preserve that while still deleting the files at the end of the run?